lib/fbe/octo.rb

```diff
@@ -109,7 +109,7 @@ def Fbe.octo(options: $options, global: $global, loog: $loog)
             builder.use(Octokit::Response::RaiseError)
             builder.use(Faraday::Response::Logger, loog, formatter: Fbe::Middleware::Formatter)
             builder.use(Fbe::Middleware::RateLimit)
-            builder.use(Fbe::Middleware::Trace, trace)
+            builder.use(Fbe::Middleware::Trace, trace, ignores: [:fresh])
```

Comment:
@Yegorov is it a typo? shouldn't it be `:http_cache_trace`?

Reply:
@yegor256 ok, i add test.